solutions/tests/test_binary_to_decimal.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Test module for binary_to_decimal function.
+Contains intentionally buggy tests for debugging practice.
```

Comment:
couldn,t find the buggy test

Reply:
I guess I should just remove this comment 🤔